roles/kubernetes-apps/registry/templates/registry-proxy-psp.yml.j2

```diff
@@ -36,8 +36,8 @@ spec:
     - 'persistentVolumeClaim'
   hostNetwork: true
   hostPorts:
-  - min: 5000
-    max: 5000
+  - min: {{ registry_min_port }}
+    max: {{ registry_max_port }}
```

Comment:
Those 2 things should be `{{ registry_port }}` then we can drop the min and max options.

Reply:
Thanks I changed it to the static port